RFCs/FS-1125-print-println-functions.md

```diff
@@ -0,0 +1,69 @@
+
+The design suggestion [print and printn alongside printf and printfn](https://github.com/fsharp/fslang-suggestions/issues/1092) has been marked "approved in principle".
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/1092)
+- [x] Approved in principle
+- [ ] [Implementation](https://github.com/dotnet/fsharp/pull/FILL-ME-IN)
+- [ ] Design Review Meeting(s) with @dsyme and others invitees
+- [x] [Discussion](https://github.com/fsharp/fslang-design/discussions/675)
+
+# Summary
+
+This RFC introduces two new functions:
+- `print : string -> unit`
+- `println : string -> unit`
+
+`print` writes the exact input to stdout (analogous to `System.Console.Write`) and `println` appends a newline to the string before writing to stdout (analogous to `System.Console.WriteLine`).
+
+# Motivation
+
+These functions will simplify the user experience when working with non-formatted strings. Currently, beginners to the language are introduced to `printf` and `printfn` as the default method for writing strings to the console. However, truly understanding these functions require understanding the `TextWriterFormat` type, which is generally out-of-scope for people initially learning the language. For example, the error generated from the following snippet would confuse many beginners.
+```fs
+let str = "A string"
+printfn str // Error FS0001: The type 'string' is not compatible with the type 'Printf.TextWriterFormat<'a>'
+```
+Particularly after the introduction of interpolated strings, many users do not need the extra power of format strings. In these cases, `print` and `println` would provide a more user-friendly default for printing strings.
+
+# Detailed design
+
+This RFC would add the two functions described above to the `FSharp.Core` library to supplement the existing `printf` and `printfn` functions. The `print` and `println` functions each take a string and write that string to stdout. `println` appends a newline to the string while `print` does not. The fact that these functions take a `string` argument means that string interpolation can still be used to provide formatting and variable capture.
+
+Example code:
+
+```fsharp
+let str = "A string"
+print str // prints "A string" with no trailing newline
+println str // prints "A string" with a trailing newline
+println $"The value in str is %s{str}" // prints "The value in str is A string" with a trailing newline
+println $"%0.3f{System.Math.PI}" // prints "3.142" with a trailing newline
+```
+
+# Drawbacks
+
+The addition of these functions brings the core library to four different `print*` functions that are automatically available and an even higher number of `*print*` functions. There is some additional cognitive load to differentiating between these functions, particularly for experienced F# users who are already accustomed to `printf` and `printfn`.
+
+# Alternatives
```

Comment:
I believe `printn` was discussed as an option, might be good to list here just for the sake of completeness, even if we're probably not gonna go with `printn` as an option